src/commands/terraformls.ts

```diff
@@ -38,6 +38,12 @@ export class TerraformLSCommands implements vscode.Disposable {
 
         await config('terraform').update('languageServer.enable', false, scope);
       }),
+      vscode.commands.registerCommand('terraform.openSettingsJson', async () => {
+        // this opens the default settings window (either UI or json)
+        await vscode.commands.executeCommand('workbench.action.openSettings', {
```

Comment:
This opens the settings for me, but jumping to the entry doesn't seem to work.

Reply:
Passing the string as second argument pre-fills the settings search, but I'm not sure if that's the solution you're looking for?
```ts
await vscode.commands.executeCommand('workbench.action.openSettings', 'terraform.languageServer.enable');
```